grid/communication/send-email/maps/sendgrid.suma

```diff
@@ -21,6 +22,16 @@ map SendEmail {
     content = content.concat([{ type: 'text/html', value: input.html }])
   }
 
+  set if (input.attachments) {
+    attachments = input.attachments.map(attachment => {
+      return {
```

Comment:
Map seem unnecessary, as it maps 1:1 input to what needs to be in request.

Reply:
I am not 100% sure how profile-validation works on input, but I _think_ it let's you keep additional object properties, which would be passed to the provider without explicit mapping. Also this marks a clear extension point if the profile changes, so personally I am in favor of having explicit mappings like this.